pyproject.toml

```diff
@@ -20,15 +20,15 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.10,<3.11"
 numpy = "^1.20.0"
-numba = "^0.55"
-pystencils = "^1.0"
+numba = ">=0.55"
 scipy = "^1.7.3"
 sympy = "^1.9"
-pyfftw = "^0.13.0"
 psutil = "^5.9.0"
 h5py = "^3.6"
 pyelastica = "^0.3"
-matplotlib = "^3.3.2"
+matplotlib = "3.7.1"
```

Comment:
Any particular reason why do we fix matplotlib version?

Reply:
@sy-cui ?

Reply:
Matplotlib 3.8.0 seems to have removed some functionalities used in sopht. Also on my machine installing any other version with the in-environment poetry would also install `setuptools-scm`, which is unwanted and keeps complaining about not having a section in `pyproject.toml`.